.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: Prepare release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Get version
+        id: version
+        uses: home-assistant/actions/helpers/version@master
```

Comment:
be careful with this one, it can change without warning <https://github.com/home-assistant/actions#helpers>

Reply:
I could go for 
```
sed -i 's|"version": "1.0.0"|"version": "${{ github.ref }}"|' custom_components/youtube/manifest.json
sed -i 's|refs/heads/||' custom_components/youtube/manifest.json
sed -i 's|refs/tags/||' custom_components/youtube/manifest.json
```
instead if you prefer - to remove the dependancy on the home assistant helper.
Let me know which you prefer

Reply:
The ref includes /refs/tag/ as a prefix, so that will not work.
It should be fine to use, but don't do it without being aware that it might change.

Reply:
I have used this previously, and it is still on my calendarific integration (https://github.com/pinkywafer/Calendarific/blob/master/.github/workflows/main.yml)

The first `sed` inserts the compete ref, and then the second two delete the unneeded parts out

in any case, I'll leave it as it is for now.  I am watching the homeassistant/actions repo so should be able to keep an eye out for any changes

Reply:
:joy: I only saw the first line, yeah that works :+1: